dashboard/app/controllers/projects_controller.rb

```diff
@@ -133,10 +133,61 @@ def public
     end
   end
 
+  def project_and_featured_project_fields
+    [
+      :storage_apps__id___id,
+      :storage_apps__storage_id___storage_id,
+      :storage_apps__value___value,
+      :storage_apps__project_type___project_type,
+      :storage_apps__published_at___published_at,
+      :featured_projects__featured_at___featured_at,
+      :featured_projects__unfeatured_at___unfeatured_at
+    ]
+  end
+
+  def combine_projects_and_featured_projects_data
+    storage_apps = "pegasus_#{CDO.rack_env}__storage_apps".to_sym
+    project_featured_project_combo_data = DASHBOARD_DB[:featured_projects].select(*project_and_featured_project_fields).
+    join(storage_apps, id: :storage_app_id).map
+    extract_data_for_tables(project_featured_project_combo_data)
+  end
+
+  def extract_data_for_tables(project_featured_project_combo_data)
+    @featured_project_table_rows = []
+    project_featured_project_combo_data.each do |project_details|
+      project_details_value = JSON.parse(project_details[:value])
+      channel = storage_encrypt_channel_id(project_details[:storage_id], project_details[:id])
+      featured_project_row = {
+        projectName: project_details_value['name'],
+        channel: channel,
+        type: project_details_value['projectType'],
```

Comment:
`type` should come from the `projectType` field, not from the `value` field. the goal is for the redundant fields to eventually be removed from `value`.

Reply:
✅